MAUI/Polar-Charts/Getting-Started.md

```diff
@@ -0,0 +1,497 @@
+---
+layout: post
+title: Getting started with .NET MAUI Chart control | Syncfusion
+description: This section explains about the getting started with Syncfusion .NET MAUI Chart (SfPolarChart) control.
+platform: maui
+control: SfPolarChart
+documentation: ug
+---
+
+# Getting Started with .NET MAUI Chart
+
+This section explains how to populate the Polar chart with data, a title, data labels, a legend, tooltips and markers. It also covers the essential aspects of getting started with the chart.
+
+## Creating an application with .NET MAUI chart
+
+1. Create a new .NET MAUI application in Visual Studio.
+2. Syncfusion .NET MAUI components are available on [nuget.org](https://www.nuget.org/). To add SfPolarChart to your project, open the NuGet package manager in Visual Studio, search for Syncfusion.Maui.Charts, and then install it.
+3. To initialize the control, import the Chart namespace.
+4. Initialize the [SfPolarChart]().
+
+{% tabs %} 
+
+{% highlight xaml %}
+
+    <ContentPage
+        . . .    
+        xmlns:chart="clr-namespace:Syncfusion.Maui.Charts;assembly=Syncfusion.Maui.Charts">
+        <Grid>
+            <chart:SfPolarChart/>
+        </Grid>
+    </ContentPage>
+    
+{% endhighlight %}
+
+{% highlight C# %}
+
+    using Syncfusion.Maui.Charts;
+    namespace ChartGettingStarted
+    {
+        public partial class MainPage : ContentPage
+        {
+            public MainPage()
+            {
+                InitializeComponent();           
+                SfPolarChart chart = new SfPolarChart(); 
+            }
+        }   
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Register the handler
+
+Syncfusion.Maui.Core NuGet package is a dependent package for all Syncfusion controls in .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.
```

Comment:
The Syncfusion.Maui.Core NuGet package is a dependent package for all Syncfusion controls in .NET MAUI. In the MauiProgram.cs file, register the handler for Syncfusion core.